main.go

```diff
@@ -115,15 +128,16 @@ func main() {
 }
 
 func segment(s *bytes.Buffer) string {
-	b := bytes.NewBufferString("\"\" +\n")
-	for s.Len() > 0 {
-		v := string(s.Next(100))
-		b.WriteString(fmt.Sprintf("\t\t\t%q", v))
-		if s.Len() > 0 {
-			b.WriteString(" +\n")
-		}
+	var b bytes.Buffer
+	b64 := base64.NewEncoder(base64.StdEncoding, &b)
+	b64.Write(s.Bytes())
+	b64.Close()
+	res := "`\n"
+	chunk := make([]byte, 76)
+	for n, _ := b.Read(chunk); n > 0; n, _ = b.Read(chunk) {
+		res += string(chunk[0:n]) + "\n"
```

Comment:
You are injecting `\n` characters into the base64-encoded string. The newlines in the multiline string with the backticks exist in the actual data now.